src/jdk.jartool/share/man/jar.md

```diff
@@ -220,20 +218,25 @@ operation modes:
 :   Prints the program version.
 
 ## Integrity of a jar Archive
-As a jar archive is based on ZIP format, it is possible to manufacture a jar archive using tools
-other than the `jar` command. The `--validate` options checks a jar archive for some integrity
-straights:
-
--   Entries in the centrail directory should be consistent with the local file headers.
--   Same names entry should only have one copy. Duplicated entries in a jar file could lead to
-    override desired content.
--   Entries names should be valid. An entry name should not:
-    - contain a drive or device letter,
-    - contain a leading slash
-    - contain backwards slashes '\'
-    - have any name element is "." or ".."
--   The API exported by a multi-release jar archive is consistent across all different release
-    versions.
+As a jar archive is based on ZIP format, it is possible to create a jar archive using tools
+other than the `jar` command. The `--validate` option performs the following integrity checks:
+
+- That there are no duplicate Zip Entry file names
+- Verify that the Zip Entry file name:
+    - is not an absolute path
+    - the file name is not '.' or '..'
+    - does not contain a backslash, '\\'
+    - does not contain a drive letter
+    - path element does not include '.' or '..
+- The API exported by a multi-release jar archive is consistent across all different release
+  versions.
+
+The jar tool will return a status code of 0 if there were no integrity issues encountered and a
+status code of 1 an issue was found. When an integrity issue is reported, it will often require
+that the jar file is re-created by the original source of the jar file.
+
+Check with the developer to ensure the jar archive integrity when warnings observed after using
+the `--validate` option.
```

Comment:
I don't see a need for this last sentence given the last sentence in the paragraph above.

I would not know who "the developer" is (though the same could be said of "the original source of the jar file"